### ISSUE TITLE
eglCompositorSwapPolicyEXT mismatch between header and description

### ISSUE THREAD

**Issue by hallershahaf** [KhronosGroup/EGL-Registry#176]
The APIs signature states that it receives external_win_id as input - 

> EGLBoolean eglCompositorSwapPolicyEXT(EGLint **_external_win_id_**, EGLint policy);

While the API description itself mentions external_ref_id as input - 

> This function shall be called by the primary context to specify the handling of buffer swaps of the context specified by **_external_ref_id_**.

**Comment by stonesthrow** [comment]
You are referring to extension: https://registry.khronos.org/EGL/extensions/EXT/EGL_EXT_compositor.txt
Lets see if the original authors are available and can respond and/or fix.

It does look like a copy and paste error. In the description for that function.
@shadazar , Are you available to respond?

**Comment by shadazar** [comment]
Yes it's a bug, we'll get the typo addressed in the near future.
